cli/src/api/pcluster_api.py

```diff
@@ -74,13 +74,24 @@ class ImageInfo:
     """Minimal representation of a building image."""
 
     def __init__(self, imagebuilder: ImageBuilder):
-        self.image_name = imagebuilder.stack.name
-        self.imagebuild_status = imagebuilder.imagebuild_status
+        # imagebuilder stack info
```

Comment:
I'd rename this class from ImageInfo to PclusterImageInfo to avoid name overlapping with ImageInfo in aws_resources.

Or ImageBuilderInfo given that you're returing both info about the ami and the image-builder stack

Reply:
good point, rename to ImageBuilderInfo